### ISSUE TITLE
tide frontend does not respect hidden_repos

### ISSUE THREAD

**Issue by kargakis** [kubernetes/test-infra#6694]
It probably should.

@BenTheElder @qhuynh96 @cjwagner @stevekuznetsov 

/area prow

**Comment by BenTheElder** [comment]
Yes. Thankfully we don't have it pointed at any currently but this is
definitely a bug...

**Comment by cjwagner** [comment]
Yeah we should change that so we can use Tide on hidden repos.
Plugin help could also contain config info related to hidden repos, but given that all that info is publicly available in `config.yaml` I don't think we need to hide it.

**Comment by kargakis** [comment]
Yeah, plugin help is fine as is. I can take a look at fixing this tomorrow

/assign
